modules/dxp/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -1,70 +1,64 @@
-@import 'atlas-variables';
-@import 'mixins';
-
 @import './click-goal';
```

Comment:
If you move this import under `cadmin-variables`, you don't need to import it in `_click-goal.scss`. Sass takes that partial and inserts it where you're importing it in `main.scss`. 

Not trying to confuse, but the only time you would need to `@import 'cadmin-variables';` in `_click-goal.scss` is if you remove the leading underscore from the file name (`click-goal.scss`). Sass will use that file in `main.scss` and also output a separate file `click-goal.css`. When Sass generates that standalone file it will look for cadmin variables and clay css mixins and not have access to it.